src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
```

Comment:
Consider generating random values for the side length in a separate class like 'FigureSupplier', rather than passing the value directly to the constructor. This will make the 'Square' class more flexible and the code more readable.